integration-tests/baas-test-server/README.md

```diff
@@ -14,6 +14,8 @@ AWS_ACCESS_KEY_ID=...
 AWS_SECRET_ACCESS_KEY=...
 ```
 
+Be sure to have `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) downloaded and added to your PATH. More info about where to get it can be found in the BAAS repo.  
```

Comment:
Perhaps we should like directly to their onboarding.md - there might be other prerequisites that need to be setup?

```suggestion
Be sure to have `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) downloaded and added to your PATH. More info about where to get it can be found [in the BAAS repo](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md).
```

Reply:
Makes sense